lib/Babel.js

```diff
@@ -32,27 +33,24 @@ function Babel(uri, callback) {
         self.combineName = query.combineName;
 
         checkType(query, 'defaultLanguage', 'string');
-        if (query.languageMap) {
-          if (typeof query.languageMap === 'string') {
-            query.languageMap = JSON.parse(fs.readFileSync(query.languageMap, 'utf8'));
-          }
-          if (typeof query.languageMap === 'object' &&
-            !Array.isArray(query.languageMap)
-          ) {
-            for (const lang of Object.keys(query.languageMap)) {
-              const value = { languageMap: query.languageMap[lang] };
-              checkType(value, 'languageMap', 'string-array', true);
-              value.languageMap.unshift(lang);
-              query.languageMap[lang] = value.languageMap;
-            }
-          } else {
-            throw new Err('languageMap must be a dictionary of' +
-              ' langCodes => [list of langs codes]');
+        query.languageMap = query.languageMap || path.join(__dirname, 'fallbacks.json');
```

Comment:
you don't really need to join with __dirname -- it will be used that way automatically.

Reply:
P.S. If you want to get absolute path from a filename, just use `path.resolve(filename)`

Reply:
We had to give it an absolute path, otherwise it doesn't work when pulled through tilerator, and it warns that the file isn't found.

Reply:
path.resolve('fallbacks.json') gives me 1 up folder. I could do path.resolve('lib/fallbacks.json') but then the join is better for the slashes...

Reply:
ah, gotcha, sorry, I was thinking of current dir, not the __dirname. My mistake. I usually prefer to use `resolve` in general, but either will work.